### DISCUSSION TITLE
validation and installation

### DISCUSSION THREAD

**Discussion by BYOMKESHPRATIHARI** [twbs/bootstrap#39009] [General]
how to start in case of beginner?

**Comment by nkdas91** [comment]
Hi @BYOMKESHPRATIHARI 

Start with the Quick Start example: https://getbootstrap.com/docs/5.3/getting-started/introduction/#quick-start
Then try out components: https://getbootstrap.com/docs/5.3/components/accordion/
Then Forms: https://getbootstrap.com/docs/5.3/forms/overview/

If you want an easier option: try w3schools: https://www.w3schools.com/bootstrap5/index.php
